spring/src/main/java/io/seata/spring/annotation/GlobalTransactionalInterceptor.java

```diff
@@ -74,6 +74,37 @@
     private static ScheduledThreadPoolExecutor executor =
         new ScheduledThreadPoolExecutor(1, new NamedThreadFactory("degradeCheckWorker", 1, true));
 
+    //region GLOBAL_TRANSACTION_TIMEOUT
+
+    private static int GLOBAL_TRANSACTION_TIMEOUT = 0;
+    private static int MIN_GLOBAL_TRANSACTION_TIMEOUT = 0;
+
+    private static void initGlobalTransactionTimeout() {
```

Comment:
remove static ？

Reply:
Consistent with the globalTransactionTimeout.